BetterLabel/BetterAttributedLabel.swift

```diff
@@ -20,7 +20,12 @@ open class BetterAttributedLabel: UIView {
     open var contentInset = UIEdgeInsets.zero {
         didSet { updateLayout() }
     }
-    
+
+    open override var backgroundColor: UIColor? {
+        get { label.backgroundColor }
+        set { label.backgroundColor = newValue }
```

Comment:
I think, that as you can have customized `contentInset` it should also be set to `self`, not just the `label`.

Reply:
Nice catch, added.

Reply:
My bad, should be rather `super` not `self` 😬 otherwise we have infinite recursion 😬

Reply:
🙈